packages/components/src/Components/TagModal/TagModal.js

```diff
@@ -1,15 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import './tagModal.scss';
-import { Modal } from '@patternfly/react-core';
+import {
+    Modal,
+    Pagination,
+    Bullseye,
+    EmptyState,
+    EmptyStateVariant,
+    Title,
+    EmptyStateBody,
+    Button
+} from '@patternfly/react-core';
 import classNames from 'classnames';
 import {
     Table,
     TableHeader,
     TableBody
 } from '@patternfly/react-table';
+import { EmptyTable } from '../EmptyTable';
+import { TableToolbar } from '../TableToolbar';
+import { PrimaryToolbar } from '../PrimaryToolbar';
+import { Skeleton } from '../Skeleton';
+import { SkeletonTable } from '../SkeletonTable';
+
+const calculateChecked = (rows = [], selected) => (
+    rows.every(({ id }) => selected && selected.find(({ id: selectedId }) => selectedId === id))
+        ? rows.length > 0
+        : rows.some(({ id }) => selected && selected.find(({ id: selectedId }) => selectedId === id)) && null
+);
+
+const unique = (arr) => (
+    arr.filter(({ id }, index, arr) => arr.findIndex(({ id: currId }) => currId === id) === index)
+);
 
 export default class TagModal extends React.Component {
+    onSelect = (_event, isSelected, rowId) => {
```

Comment:
if not using _event descrust as follows 🤔 ? `onSelect = ({isSelected, rowId}) => {`